stdlib/planner/group_first_last_test.flux

```diff
@@ -235,13 +234,15 @@ testcase group_one_tag_first {
         testing.load(tables: inData)
             |> range(start: -100y)
             |> group(columns: ["t0"])
+            // Sort to make comparison more reliable when using storage (where
+            // row ordering can vary).
+            |> sort(columns: ["_time", "_field", "t0", "t1"])
```

Comment:
I'm still not clear if the sort added to the tests in this file will impact the pushdowns applied.

I've got a concern about losing coverage, but the tradeoff here is we'd need to rework the `inData` more deliberately to give more deterministic answers when storage is used. Since `inData` is shared by _all the tests_ in this file, I opted to _not do that_ for this PR.

Reply:
Yes, it does affect the pushdowns applied, for OSS. I filed this issue:
https://github.com/influxdata/influxdb/issues/23757